src/templates/CheckOut/CheckOutPartials.js

```diff
@@ -80,10 +74,16 @@ export default {
 
     return (
       <Col className={styles['main-sidebar']}>
-        <div className='sidebar'>{children}</div>
```

Comment:
remove line 63 `expect(wrapper.find('.sidebar')).to.have.length(1);` in checkout.spec.js